style/style.less

```diff
@@ -0,0 +1,2 @@
+@import './popup';
```

Comment:
I like this change but `index` should probably be the one that includes the others.